packages/block-library/src/site-title/edit.js

```diff
@@ -123,16 +123,16 @@ export default function SiteTitleEdit( {
 					label={ __( 'Settings' ) }
 					resetAll={ () => {
 						setAttributes( {
-							isLink: false,
+							isLink: true,
 							linkTarget: '_self',
 						} );
 					} }
 					dropdownMenuProps={ dropdownMenuProps }
 				>
 					<ToolsPanelItem
-						hasValue={ () => isLink !== false }
+						hasValue={ () => isLink === false }
```

Comment:
Nit: Since `isLink` is a Boolean type, we should be able to write it like this:

```suggestion
						hasValue={ () => ! isLink }

```

Reply:
I've updated the PR to reflect the suggested change.